### ISSUE TITLE
Lost my data after update

### ISSUE THREAD

**Issue by MatKylo** [SchizoDuckie/DuckieTV#953]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Chrome extension

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
V1.1.5

**What is your Operating System (Windows, Mac, Linux, Android)**
Windows 10

**Describe the problem you are having and steps to reproduce if available**
I just log in the app and after a message of a new update and news features I just lost all of my backup.
To add to that the tab "view your favorite" is not working anymore, just showing a loading message on top. what can I do to debug it?

**Comment by garfield69** [comment]
Probably best if I take a look at the database and settings directly.      
Could you:     
-  locate `C:\Users\<username>\AppData\Local\Google\Chrome\User Data\<profile>` and zip (with subdirs enabled)     
(`<profile>` is usually `default` but may also be your chrome username)     
- upload the zip archive to a cloud drive such as [Google Drive](https://www.google.com/drive/) for instance and post a share link below

Once I've taken a look I can suggest appropriate recovery methods ;-)      

[edit] might want to quit the chrome browser while making the zip, to avoid dropping locked files.      

[edit2] if the zip is too big, then all I really need are the following two folders:     
```
C:\Users\<username>\AppData\Local\Google\Chrome\User Data\<profile>\databases
C:\Users\<username>\AppData\Local\Google\Chrome\User Data\<profile>\Local Storage
```

**Comment by SFNMutt** [comment]
I have this same problem.  i'm using the browser action chrome extension & all my shows are gone after the 1.15 update.  on a 2nd computer w/ the same extensions installed, Duckie is still working so I was able to grab a backup.  I installed the backup, added a couple new shows, closed Duckie, & then opened it back up only to find all shows gone again.

I disabled all other extensions but no change.

I uninstalled & reinstalled the extension then imported the backup.  that seems to have fixed it.

**Comment by garfield69** [comment]
@MatKylo  so were you able to resolve this?       
Several other users have had similar problems, and for those without a DuckieTV *.backup or a Trakt account to sync from, I was able to recover their data from the database and settings files they sent me via a cloud drive share-link.

**Comment by MatKylo** [comment]
Hi, 


I was able to correct the problem. I had to delete the app and install it back to get a functional version. After that, it just take me some time to build back my show list. 
Thanks. 




----- Mail original -----

De: "garfield69" <notifications@github.com> 
À: "SchizoDuckie/DuckieTV" <DuckieTV@noreply.github.com> 
Cc: "MatKylo" <calot.mathieu@free.fr>, "Mention" <mention@noreply.github.com> 
Envoyé: Mercredi 4 Octobre 2017 23:49:17 
Objet: Re: [SchizoDuckie/DuckieTV] Lost my data after update (#953) 


@MatKylo so were you able to resolve this? 
Several other users have had similar problems, and for those without a DuckieTV *.backup or a Trakt account to sync from, I was able to recover their data from the database and settings files they sent me via a cloud drive share-link.
